exercises/zipper/canonical-data.json

```diff
@@ -0,0 +1,715 @@
+{
+  "exercise": "zipper",
+  "version": "1.0.0",
+  "comments": [
+    " The test cases for this exercise include an initial tree and a     ",
+    " series of operations to perform on the initial tree.               ",
+    "                                                                    ",
+    " Trees are encoded as nested objects. Each node in the tree has     ",
+    " three members: 'value', 'left', and 'right'. Each value is a       ",
+    " number (for simplicity). Left and right are trees. An empty node   ",
+    " is encoded as null.                                                ",
+    "                                                                    ",
+    " Each operation in the operations list is an object. The function   ",
+    " name is listed under 'operation'. If the function requires         ",
+    " arguments, the argument is listed under 'item'. Some functions     ",
+    " require values (i.e.  numbers), while others require trees.        ",
+    " Comments are always optional and can be used almost anywhere.      "
+  ],
+  "cases": [
+    {
+      "description": "Creating a zipper for a binary tree",
+      "cases": [
```

Comment:
far as I can tell the only reason for this second level of nesting under `cases` is to attach a description to this entire group? I feel like we could do without, and just one level of nesting under `cases`. Yes? No?

Reply:
This...was a mistake. I think I copied another exercise as a starting point and didn't realize this was happening. I just changed it.